code/game/objects/effects/spiders.dm

```diff
@@ -37,16 +38,37 @@
 
 /obj/structure/spider/stickyweb/CanAllowThrough(atom/movable/mover, turf/target)
 	. = ..()
-	if(istype(mover, /mob/living/simple_animal/hostile/poison/giant_spider))
+	if(!genetic)

```

Comment:
```DM
if(genetic)
    return
```

Reply:
you don't even need this var though, you've already subtyped it?

Reply:
definitely more readable